src/shared/orgListUtil.ts

```diff
@@ -341,4 +338,36 @@ export class OrgListUtil {
   }
 }
 
-export const identifyActiveOrgByStatus = (org: ExtendedAuthFields): boolean => org.status === 'Active';
+export const identifyActiveOrgByStatus = (org: FullyPopulatedScratchOrgFields): boolean =>
+  'status' in org && org.status === 'Active';
+
+/** Identify the default orgs */
+const identifyDefaultOrgs = (
+  orgInfo: AuthFieldsFromFS,
+  config: JsonMap
+): ExtendedAuthFields | ExtendedAuthFieldsScratch => ({
+  ...orgInfo,
+  ...((config['target-org'] && (orgInfo.username === config['target-org'] || orgInfo.alias === config['target-org'])) ||
```

Comment:
@mshanemc can/will this be simplified by asking the config aggregator for default target-org and dev-hub?

Reply:
https://github.com/salesforcecli/plugin-org/pull/550/commits/8989ccab2ba3ce247738b57a2afdf5fcf7783123